swagger.json

```diff
@@ -514,6 +577,80 @@
           "example": "https://gravatar.com/avatar/aed61baf1e9256ed7d70e2cbbfcba9aa?s=400&d=robohash&r=x"
         }
       }
+    }, "update": {
+      "type": "object",
+      "required": [
```

Comment:
These properties _email_, _firstName_, _lastName_, _password_ are optional. They shouldn't be required compulsorily. A user should be able to include only firstName OR email OR lastName. They don't have to provide all at once